### PR TITLE
avoid importing order components from other libs

### LINKED ISSUE #14099: Remove order components import from organization lib and checkout lib

### PR DESCRIPTION
**Pull request by WeizhengSap**
close #14099

### CODE REVIEW

File: feature-libs/cart/saved-cart/components/page-context/saved-cart-details-page/saved-cart-order-entries-context.ts [hunk 1]
```diff
@@ -1,12 +1,10 @@
 import { Injectable } from '@angular/core';
 import { ActionsSubject } from '@ngrx/store';
+import { CartOrderEntriesContext } from '@spartacus/cart/main/components';
```
Comment (plabadie) [inline feature-libs/cart/saved-cart/components/page-context/saved-cart-details-page/saved-cart-order-entries-context.ts]: What do we do about this import from '@spartacus/cart/main/components' ?

File: feature-libs/cart/saved-cart/components/page-context/saved-carts-page/new-saved-cart-order-entries-context.ts [hunk 2]
```diff
@@ -1,10 +1,8 @@
 import { Injectable } from '@angular/core';
 import { ActionsSubject } from '@ngrx/store';
+import { CartOrderEntriesContext } from '@spartacus/cart/main/components';
```
Comment (plabadie) [inline feature-libs/cart/saved-cart/components/page-context/saved-carts-page/new-saved-cart-order-entries-context.ts]: What do we do about this import from '@spartacus/cart/main/components' ?

**Review by plabadie — COMMENTED** [review]
Let's discuss this PR

File: feature-libs/cart/saved-cart/components/page-context/saved-carts-page/new-saved-cart-order-entries-context.ts (hunk 2 above)
Comment (WeizhengSap) [reply feature-libs/cart/saved-cart/components/page-context/saved-carts-page/new-saved-cart-order-entries-context.ts]: this ticket only remove the imports of `order/components`

**Pull request closed without merging**